src/lucky/renderable.cr

```diff
@@ -96,6 +96,22 @@ module Lucky::Renderable
     end
   end
 
+  private def file(path,
+                   content_type : String? = nil,
+                   disposition : String = "attachment",
+                   filename : String? = nil,
+                   status : Int32? = nil)
+    Lucky::FileResponse.new(context, path, content_type, disposition, filename, status)
+  end
+
+  private def file(path,
+                   content_type : String? = nil,
+                   disposition : String = "attachment",
+                   filename : String? = nil,
+                   status : Lucky::Action::Status = Lucky::Action::Status::OK)
+    Lucky::FileResponse.new(context, path, content_type, disposition, filename, status.value)
```

Comment:
Instead of instantiating the `FileResponse` here, what do you think about calling the other `file` method. That way there is just one method that is the source of all truth. Right now it probably doesn't buy us much to do this, but in the future it might be helpful.